src/components/cookie-banner/index.md.njk

```diff
@@ -17,121 +17,153 @@ Allow users to accept or reject cookies which are not essential to making your s
 
-{{ example({group: "components", item: "cookie-banner", example: "no-js", html: true, nunjucks: true, open: false}) }}
+Keep the text as short as possible while making sure it’s an accurate description of how you use cookies. For example, if you use more than one ‘functional’ cookie and there’s not enough space to mention what each of them does, you could ask for permission to set cookies so ‘you can use as many of the service’s features as possible’.
 
-If your service only uses essential cookies, display a different version of the cookie banner to users whose devices are not running JavaScript.
+[See the cookies page for more advice on writing about cookies](/patterns/cookies-page/)].
 
-{{ example({group: "components", item: "cookie-banner", example: "essential-no-js", html: true, nunjucks: true, open: false}) }}
+### If you’re using essential cookies and analytics cookies
 
-## Getting consent from users who are not running JavaScript
+You can use this example text for a service which sets essential and analytics cookies. Analytics cookies are those set by your organisation to collect information about how people are using your digital service.
 
-[Some users will not be running JavaScript on their devices](https://gds.blog.gov.uk/2013/10/21/how-many-people-are-missing-out-on-javascript-enhancement/). If you’re using non-essential cookies and want to get consent from these users, you’ll need an approach that does not depend on JavaScript.
+{{ example({group: "components", item: "cookie-banner", example: "default", html: true, nunjucks: true, open: false}) }}
 
-Show the replacement message confirming that the user has accepted or rejected cookies when they submit the form and the page has reloaded.
+### If you’re using more than one type of non-essential cookie
 
-Here’s an example showing how it should work.
+You can use this example text for a service that set:
+- essential cookies
+- analytics cookies
+- functional cookies to remember the user’s settings but are not essential
 
-{{ example({group: "components", item: "cookie-banner", example: "server-side", html: true, nunjucks: true, open: true}) }}
+{{ example({group: "components", item: "cookie-banner", example: "multiple-cookies", html: true, nunjucks: true, open: false}) }}
 
 ## Creating a cookies page
 
-You’ll need a [cookie page](/patterns/cookies-page/) as well as a cookie banner.
+You’ll need a [cookies page](/patterns/cookies-page/) as well as a cookie banner.
 
-When the user sets or changes their cookie preferences using the controls on the cookie page (rather than in the cookie banner), display this version of the replacement message. This is so they can get back to the page they were looking at.
+When the user sets or changes their cookie preferences using the controls on the cookies page (rather than in the cookie banner), display this version of the confirmation message. This is so they can get back to the page they were looking at.
```

Comment:
Move this instruction about the confirmation message (technically a notification banner) into the cookies page pattern.
```suggestion
```